docs/source/user-guide/tutorials/building-conda-packages.rst

```diff
@@ -158,17 +150,16 @@ Installing only one makes it easier to keep track of
 the builds, but it is possible to have both installed on the same
 system at the same time. If you have both installed, use the
 ``where`` command on Windows, or ``which`` command on Linux to
-see which version comes first on PATH since this is the one you will be using::
+see which version comes first on ``PATH`` since this is the one you will be using::
```

Comment:
Won't let me suggest on a non-edited line, but could we do:

"If you have both installed, use the ``where`` command on Windows, or ``which`` command on macOS or Linux, to see which version comes first on ``PATH``, since this is the one you will be using"